org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
```

Comment:
if it support wayland then it's `fallback-x11` and not `x11`. Otherwise remove wayland.

Reply:
ok removed wayland, it seems that X11 socket is enough.